src/Xamarin.Android.Build.Tasks/Utilities/ManifestDocument.cs

```diff
@@ -43,6 +43,22 @@ internal class ManifestDocument
 			{ "action",   "android.intent.action.MAIN" },
 			{ "category", "android.intent.category.LAUNCHER" },
 		};
+
+		// the elements which we apply the "." -> PackageName replacement on
+		static readonly string [] ManifestElementFixupNames = {
```

Comment:
Silly alternate idea: Do we want attributes to be "global" to all elements, or "local" to specific elements?

If we actually want local, perhaps a `Dictionary<string, string[]>` would be better?

```csharp
static readonly Dictionary<string, string[]> ManifestElementFixupNames = new Dictionary<string, string[]>() {
    ["activity"] = new string[]{
        "name",
        // ...
    },
    ["service"] = new string[]{
        "name",
        // ...
    },
};
```